package.json

```diff
@@ -100,6 +100,7 @@
     "react-test-renderer": "16.2.0",
     "react-tooltip": "3.8.0",
     "react-virtualized": "9.20.1",
+    "react-visibility-sensor": "^5.0.2",
```

Comment:
@cwillisf I think we usually pin user-facing dependencies to prevent untraceable update issues. (i.e. without a package-lock, it isn't possible to have reproducible builds without pinning to specific version)